geos_perf.c

```diff
@@ -287,9 +314,15 @@ int
 main(int argc, char *argv[])
 {
     initGEOS(geos_log_stderr, geos_log_stderr);
-
     log_stderr("VERSION [GEOS %s]\n", GEOSversion());
 
+    geos_lib_handle = dlopen("libgeos_c.so", RTLD_LAZY);
```

Comment:
Hm, will need to do something about file extensions, "libgeos_c.dylib" is what I've got. There is a `dlopen()` documented so that part might work. I will experiemnt.